File: Practica_1/src/gestorAplicacion/modelos/Persona.java
package gestorAplicacion;

import java.io.Serializable;

/*Clase  abstracta creada para la herencia de Usuario y empleado*/
abstract public class Persona implements Serializable{
private static final long serialVersionUID = 1L;
    protected String nombre;
    protected int identificacion;
    protected int telefono;

    // METODOS
    /* Metodo a heredar que muestra la imformacion importante del objeto */
    public abstract String informacion();

    // CONSTRUCTOR
    public Persona(String nombre, int identificacion, int telefono) {
        this.nombre = nombre;
        this.identificacion = identificacion;
        this.telefono = telefono;
    }

    // GETTERS AND SETTERS
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getIdentificacion() {
        return identificacion;
    }

    public void setIdentificacion(int identificacion) {
        this.identificacion = identificacion;
    }

    public int getTelefono() {
        return telefono;
    }

    public void setTelefono(int telefono) {
        this.telefono = telefono;
    }

}

File: Python/gestorAplicacion/Persona.py
import abc

class Persona(abc.ABC):
    def __init__(self, nombre, identificacion, telefono):
        self.nombre = nombre
        self.identificacion = identificacion
        self.telefono = telefono

    @abc.abstractmethod
    def informacion(self):
        pass

    def get_nombre(self):
        return self.nombre

    def set_nombre(self, nombre):
        self.nombre = nombre

    def get_identificacion(self):
        return self.identificacion

    def set_identificacion(self, identificacion):
        self.identificacion = identificacion

    def get_telefono(self):
        return self.telefono

    def set_telefono(self, telefono):
        self.telefono = telefono
